### ISSUE TITLE
Cant turn off ground searching

### ISSUE THREAD

**Issue by phoenixman1000** [amblelabs/ait#812]
**Describe the bug**
there used to be (1.1.0 and before) a option on land type called "NONE" which would disable ground searching. Now there doesnt seem to be one. If you left click it it engages/disengages horizontal searching, but it doesnt fix it

**To Reproduce**
1. enter a tardis
2. Click on land type

**Expected behavior**
there should be a option "NONE" (i think)

**Screenshots**
![image](https://github.com/user-attachments/assets/38af6ffa-b913-4d43-95c3-7041252e10ec)
![image](https://github.com/user-attachments/assets/ec116a93-4db0-4bc8-b072-8bb3c455c6dc)
![image](https://github.com/user-attachments/assets/6dddef92-e477-43de-90b3-dd8c5dbf0b4c)

**AIT Version:**
ait-1.2.0.1143-1.20.1-beta.jar

**Comment by J0riUnderscore** [comment]
yeah I think I've kind of got this too. in one of my worlds I tried to set the Y cord but it wouldn't change, messing with the land type control wouldn't fix this either.

**Comment by phoenixman1000** [comment]
literally the same thing happened to me!

**Comment by Saturnorsomthing** [comment]
fixed 
![cat](https://github.com/user-attachments/assets/e1c167b1-d668-447d-85b7-bae8b016051b)
